requirements.txt

```diff
@@ -4,3 +4,5 @@ semantic_version
 twine
 wheel
 PyJWT
+gitchangelog
+pystache
```

Comment:
why did you choose this library? isn't jinja2 more wide-spread?

Reply:
Off course jinja is more wide-spread and better! I had to add that because the gitChangeLog only supports mustache and mako. The Mustache is powered by the pystache mentioned [here](https://github.com/vaab/gitchangelog/edit/master/README.rst#L262-L270)

Reply:
oki, got it